packages/providers/src/json-rpc-provider.ts

```diff
@@ -80,26 +81,37 @@ export class JsonRpcProvider extends Provider {
         return this.sendJsonRpc('status', []);
     }
 
+    /**
+     * Sends a signed transaction to the RPC
+     *
+     * @param signedTransaction The signed transaction being sent
+     * @param waitUntil
+     */
+    async sendTx(signedTransaction: SignedTransaction, waitUntil: TxExecutionStatus): Promise<TxOutcome> {
+        const bytes = encodeTransaction(signedTransaction);
+        return this.sendJsonRpc('send_tx', { signed_transaction: Buffer.from(bytes).toString('base64'), wait_until: waitUntil });
+    }
```

Comment:
`sendTx` may not be the best name. The existing ones are:
- `sendTransaction` which wait for the transaction to be finalized
- `sendTransactionAsync` which doesn't wait for the transaction to be finalized.

I think that `sendTransactionUntil` could be a better name

Reply:
yes, `sendTransactionUntil` is better